TypeCobol.Analysis/Cfg/ControlFlowGraphBuilder.MultiBranchContext.cs

```diff
@@ -0,0 +1,183 @@
+﻿using System.Collections.Generic;
+using TypeCobol.Analysis.Graph;
+using TypeCobol.Compiler.Nodes;
+
+namespace TypeCobol.Analysis.Cfg
+{
+    public partial class ControlFlowGraphBuilder<D>
+    {
+        /// <summary>
+        /// The context of a Multi Branch instruction like IF or Evaluate.
+        /// </summary>
+        internal class MultiBranchContext
+        {
+            /// <summary>
+            /// Current Block before multi branches
+            /// </summary>
+            internal BasicBlockForNode CurrentBlock;
```

Comment:
Same remark, this is the StartBlock

Reply:
In CFG a Start Block is in fact a Root Block, this is not always the case here. But ok, it is like the SourceBlock. or OriginBlock.

Reply:
done